### PR TITLE
Drop `--study` in `cli.py`

### PR DESCRIPTION
**Pull request by eukaryo**
## Motivation

I want to drop deprecated `--study` argument in cli interface.
This PR is related to PR https://github.com/optuna/optuna/pull/1079 and Issue https://github.com/optuna/optuna/issues/1059 .

## Description of the changes

I deleted codes related to`--study` in `cli,py`.

### CODE REVIEW

**Comment by HideakiImamura** [comment]
@not522 Could you review this PR?

**Review by HideakiImamura — APPROVED** [review]
LGTM.

File: optuna/cli.py [hunk 1]
```diff
@@ -362,9 +362,6 @@ class _StudySetUserAttribute(_BaseCommand):
     """Set a user attribute to a study."""
 
     def add_arguments(self, parser: ArgumentParser) -> None:
-        parser.add_argument(
-            "--study", default=None, help="This argument is deprecated. Use --study-name instead."
-        )
         parser.add_argument(
             "--study-name",
             default=None,
```
Comment (not522) [inline optuna/cli.py:367]: How about making `--study-name` required and removing if-statement in `take_action`?

```suggestion
            required=True,
```

File: optuna/cli.py [hunk 2]
```diff
@@ -376,16 +373,7 @@ def add_arguments(self, parser: ArgumentParser) -> None:
...
-            )
-        elif parsed_args.study:
-            message = "The use of `--study` is deprecated. Please use `--study-name` instead."
-            warnings.warn(message, FutureWarning)
-            study = optuna.load_study(storage=storage, study_name=parsed_args.study)
-        elif parsed_args.study_name:
+        if parsed_args.study_name:
             study = optuna.load_study(storage=storage, study_name=parsed_args.study_name)
         else:
             raise ValueError("Missing study name. Please use `--study-name`.")
```
Comment (not522) [inline optuna/cli.py:379]:
```suggestion
        study = optuna.load_study(storage=storage, study_name=parsed_args.study_name)
```

**Comment by not522** [comment]
Thank you for your PR. `--study` was deprecated in v1.5.0 and I think we can remove it.

**Review by not522 — APPROVED** [review]
LGTM

**Pull request merged**